test/screenshot/dialog/confirmation.tsx

```diff
@@ -23,33 +25,39 @@ class Confirmation extends React.Component<{}, {isOpen: boolean; action: string;
           <Button
             raised
             onClick={() => this.setState({isOpen: !this.state.isOpen})}
-          >{this.state.isOpen ? 'close dialog' : 'open dialog' }</Button>
+          >
+            {this.state.isOpen ? 'close dialog' : 'open dialog' }
+          </Button>
           <p className='mdc-typography--body1'>Dialog Action:
-            <samp>&emsp;{this.state.action}</samp>
+            <samp>&emsp;{this.state.action} &bull; Selected Index: {this.state.selectedIndex}</samp>
           </p>
         </aside>
         <Dialog
-          escapeKeyAction='esc'
-          scrimClickAction='click'
+          escapeKeyAction={''}
```

Comment:
why did you change this ?

Reply:
As I was writing the documentation I saw that if escapeKeyAction was set to an empty string it should disable closing the dialog via escape key--which sounds like an accessibility problem. Nevertheless, I wanted to *see* it in action. I could change it back it you like.

Reply:
gotcha..ya could you make add these 2 test cases in the unit tests?

Also please change it back as it provides more insight that way (than an empty string).